packages/core/styles/components/_combobox.scss

```
@@ -78,11 +56,16 @@ ix-combobox [data-chips] span {
   align-items: center;
   gap: 4px;
 
-  padding: 2px 6px;
-  border-radius: 999px;
+  padding: 0 var(--ix-space-sm);
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)